runatlantis.io/docs/automerging.md

```diff
@@ -0,0 +1,17 @@
+# Automerging
```

Comment:
Wouldn't the service account used to post comments etc. require write permissions for this to work?

Reply:
Nothing that contradicts it, but it doesn't mention that requirement explicitly anywhere :)

Reply:
It's required for Atlantis to work correctly, so I don't see why it would be explicitly called out in this section of the doc.

Reply:
See here: https://www.runatlantis.io/docs/deployment.html#create-an-access-token-for-atlantis

Reply:
Ah, cool. Nvm then 👍

Reply:
It actually does require increased permissions depending on your setup. For example, my user token can comment on public pull requests but it can't merge unless I'm a collaborator.

The token needs to be from a user with those permissions.